pkg/specgen/volumes_windows_test.go

```diff
@@ -0,0 +1,77 @@
+package specgen
```

Comment:
friendly remainder that we never run windows or macos unit tests anywhere in CI, someone must hook that up.
I mentioned to @l0rd nefore.

That is nothing against your test and we keep it but a windows maintainer must manually verify them, I don't have access to such machine so I cannot do that.

Reply:
I was able to run these tests successfully on Windows